extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -87,6 +87,7 @@ amp-lightbox-gallery .amp-carousel-button {
   width: 24px;
   height: 24px;
   padding: 16px;
+  box-sizing: content-box;
```

Comment:
I wasn't sure about this, since the intention here is to allow them to edit the dimensions of the button. There seemed to be an argument to allow them to style this however they please.

Reply:
fair point. LGTM